### PR TITLE
Add sort by track number, fix sort by title

### PR DESCRIPTION
**Pull request by ssihala**
Adds another radio option to sort by track #. Also modifies title sort such that numbers in the file's title aren't ignored

### CODE REVIEW

File: app/src/main/kotlin/com/simplemobiletools/musicplayer/models/Track.kt [hunk 1]
```diff
@@ -42,7 +39,7 @@ data class Track(
                 when {
                     title == MediaStore.UNKNOWN_STRING && other.title != MediaStore.UNKNOWN_STRING -> 1
                     title != MediaStore.UNKNOWN_STRING && other.title == MediaStore.UNKNOWN_STRING -> -1
-                    else -> AlphanumericComparator().compare(title.toLowerCase(), other.title.toLowerCase())
+                    else -> AlphanumericComparator().compare(getProperTitle(3).toLowerCase(), other.getProperTitle(3).toLowerCase())
```
Comment (tibbi) [inline app/src/main/kotlin/com/simplemobiletools/musicplayer/models/Track.kt:42]: avoid magic numbers like 3

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<resources>
+<resources xmlns:tools="http://schemas.android.com/tools">
```
Comment (tibbi) [inline app/src/main/res/values/strings.xml:2]: remove this and add new strings into all files please

Comment (ssihala) [reply app/src/main/res/values/strings.xml]: Done

File: app/src/main/kotlin/com/simplemobiletools/musicplayer/models/Track.kt (hunk 1 above)
Comment (ssihala) [reply app/src/main/kotlin/com/simplemobiletools/musicplayer/models/Track.kt]: Done

File: app/src/main/res/values-ar/strings.xml [hunk 3]
```diff
@@ -88,8 +88,9 @@
     <!-- FAQ -->
     <string name="faq_1_title">How can I fast-forward songs\?</string>
     <string name="faq_1_text">You can click on the current or max duration texts near the seekbar, that will move the song either backward, or forward.</string>
+    <string name="trackID">Track Number</string>
```
Comment (tibbi) [inline app/src/main/res/values-ar/strings.xml:91]: add it at the proper place please, not at the bottom of the file

Comment (ssihala) [reply app/src/main/res/values-ar/strings.xml]: Sorry, that was careless of me. I placed it under sorting for all the translations

**Pull request merged**